templates/repo/migrate/options.tmpl

```diff
@@ -1,13 +1,8 @@
-<div class="inline field">
+<div class="inline field" {{if .DisableNewPullMirrors}} hidden{{end}}>
```

Comment:
Instead of simply rendering this and hiding it can't we simply stick the whole div within the if block?

Reply:
Agree, better to not render it at all.

Reply:
Was hidden to avoid lack of param in backend call. Tested than disabling rendering seems to work ok so changed according to your proposal in 0d45d0c2a020f4275d7d3084471d2cffe3c80374. Please verify if lack of param won't break something in the backend.